tools/osbuild-mpp

```diff
@@ -522,7 +522,11 @@ class ManifestFile:
     @staticmethod
     def load_from_fd(f, path, overrides, default_vars):
         # We use OrderedDict to preserve key order (for python < 3.6)
-        data = json.load(f, object_pairs_hook=collections.OrderedDict)
+        try:
+            data = json.load(f, object_pairs_hook=collections.OrderedDict)
+        except json.decoder.JSONDecodeError as err:
+            print (f"Invalid json in \"{path}\": {err.msg} at line {err.lineno} (col {err.colno})")
+            sys.exit(1)
```

Comment:
Alternatively, we could just re-raise the error but this feels a nicer output than the full stacktrace

Reply:
If this was a module or something it might re-raise, but since its a CLI tool, this seems better.